plugins/tinymcespellchecker.md

```diff
@@ -112,85 +117,99 @@ tinymce.init({
-- When set to `false`, the [`spellcheck` toolbar button](#toolbarbuttons) will enable or disable as-you-type spell checking and the [`spellcheckdialog` toolbar button](#toolbarbuttons) will open the spell checker dialog.
+**Type:** comma-separated `String`
+
+**Default Value:**
+````
+'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
+````
+
+#### Example: `spellchecker_languages`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_languages: 'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
+});
+```
+
+### `spellchecker_on_load`
+
+> **Note**: Removed in Spell Checker Pro 2.0 (TinyMCE 5.2). Spell Checker Pro will now always run on editor initialization. To disable Spell Checker Pro on load, use [`spellchecker_active`](#spellchecker_active).
+
+This option runs the spellchecker when the contents of the editor is loaded.
 
 **Type:** `Boolean`
 
 **Default Value:** `false`
 
 **Possible Values:** `true`, `false`
 
-#### Example: `spellchecker_dialog`
+#### Example: `spellchecker_on_load`
 
 ```js
 tinymce.init({
   selector: 'textarea',
   plugins: 'tinymcespellchecker',
-  toolbar: 'spellchecker',
-  spellchecker_dialog: true
+  spellchecker_on_load: true
 });
 ```
 
-### `spellchecker_whitelist`
+### `spellchecker_rpc_url`
 
-This option specifies an array of words to be ignored by the spell checker.
+This option specifies the URL of the server-side `ephox-spelling` service. For instructions on how to setup a Spell Checker Pro server-side component, see: the [server-side component installation guide]({{site.baseurl}}/enterprise/server/).
 
-**Type:** `String[]`
+> **Note:** `spellchecker_rpc_url` is **not** required when enabling this plugin via [{{site.cloudname}}]({{site.baseurl}}/cloud-deployment-guide/editor-and-features/)
 
-#### Example: `spellchecker_whitelist`
+**Type:** `String`
+
+#### Example: `spellchecker_rpc_url`
 
 ```js
 tinymce.init({
   selector: 'textarea',
   plugins: 'tinymcespellchecker',
-  spellchecker_whitelist: ['tinymce','TinyMCE']
+  spellchecker_rpc_url: 'localhost/ephox-spelling'
 });
 ```
 
-### `spellchecker_on_load`
-
-> **Note**: Removed in Spell Checker Pro 2.0 (TinyMCE 5.2). Spell Checker Pro will now always run on editor initialization. To disable Spell Checker Pro on load, use [`spellchecker_active`](#spellchecker_active).
+### `spellchecker_select_languages`
 
-This option runs the spellchecker when the contents of the editor is loaded.
+This option specifies the languages that can be set for content when working with multi-language content. The specified languages will be available from the [`language` toolbar drop-down menu button](#toolbarbuttons).
```

Comment:
You might want to mention that unlike the other list of languages this one can take any language, not just the ones that the spellchecker actually supports.

Reply:
checked with @jhaines, users should only specify languages supported by spell checker pro, so I've added a link to the supported languages table on the same page.